buildSrc/src/main/kotlin/com/datadog/gradle/config/FlavorBuildConfig.kt

```diff
@@ -23,6 +24,33 @@ fun sampleAppConfig(filePath: String): SampleAppConfig {
     }
 }
 
+@Suppress("UnstableApiUsage")
+fun ApplicationDefaultConfig.configureFlavorForBenchmark(
+    rootDir: File
+) {
+    val config = sampleAppConfig("${rootDir.absolutePath}/config/benchmark.json")
+    buildConfigField(
+        "String",
+        "BM_RUM_APPLICATION_ID",
```

Comment:
minor: maybe use `benchmark` instead of `BM`, everywhere? Took me a while to understand what `BM` means.